src/routes/v2/address.ts

```diff
@@ -660,15 +660,24 @@ async function transactionsFromInsight(
   currentPage: number = 0
 ): Promise<TransactionsInterface> {
   try {
-    const path: string = `${process.env.BITCOINCOM_BASEURL}txs/?address=${thisAddress}&pageNum=${currentPage}`
+    let addr: string
+    if (
+      process.env.BITCOINCOM_BASEURL === "https://bch-insight.bitpay.com/api/"
+    ) {
+      addr = Utils.toCashAddress(thisAddress)
+    } else {
+      addr = Utils.toLegacyAddress(thisAddress)
+    }
```

Comment:
```suggestion
    const addr: string = process.env.BITCOINCOM_BASEURL === "https://bch-insight.bitpay.com/api/" ?
      Utils.toCashAddress(thisAddress) : Utils.toLegacyAddress(thisAddress)
```

Reply:
^ I am ok with both `if` or `ternary`, you can pick what you like. Just want to show and alternative (with a little plus on we can avoid using `let` here).